src/Injector.php

```diff
@@ -113,11 +107,10 @@ public function canCreate(string $name): bool
      *
      * @param string $name Class name or service alias
      * @param array  $parameters Constructor parameters, keyed by the parameter name.
-     * @return object|null
      * @throws ClassNotFoundException
      * @throws RuntimeException
      */
-    public function create(string $name, array $parameters = [])
+    public function create(string $name, array $parameters = []): ?object
```

Comment:
this is also BC break when extended class override it, ref https://3v4l.org/QTiTt